Mathlib/Algebra/Order/Star/Basic.lean

```diff
@@ -44,6 +44,19 @@ universe u
 
 variable {R : Type u}
 
+/- This lemma is placed in this file instead of `Algebra.Star.SelfAdjoint` because
+`AddSubmonoid.closure` is not available in that file.
+
+Note that if `R` is a group, this follows easily from `AddSubgroup.closure_le` applied with
+`SelfAdjoint R`. -/
```

Comment:
Is there not an alaogous proof with `AddSubmonoid.closure_le`?

Reply:
I'm suggesting defining `selfAdjointSubmonoid`

Reply:
If rather just define `selfAdjoint` as a submonoid coerced to a type instead if we're going to go down this road.

Reply:
I'm just removing this from the PR as it's not necessary (it was in an earlier version of my proof).